knack/preview.py

```diff
@@ -0,0 +1,109 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from six import string_types as STRING_TYPES
+
+from .util import ColorizedString
+
+_PREVIEW_TAG = '[Preview]'
```

Comment:
nit: my vote would be for lowercase on this, but it's a weak opinion.

Reply:
I wanted to keep it consistent with `[Deprecated]`

Reply:
Love consistency, thanks for tell me. :)